conf/messages.cy

```diff
@@ -0,0 +1,153 @@
+site.tryAgain = Try again
+site.startAgain = Start again
+site.confirm = Confirm
+site.saveAndContinue = Save and continue
+site.submitAndContinue = Submit and continue
+site.textarea.char_limit = (Limit is {0} characters)
+site.govuk = GOV.UK
+site.signOut = Sign out
+site.getHelpWithThisPage = Get help with this page
+site.notProvided = Not Provided
+site.incomplete = Incomplete
+site.change = Change
+site.confirmAndSend = Confirm and send
+site.acceptAndSend = Accept and send
+
+footer.accessibility   = Accessibility statement
+footer.cookies         = Cookies
+footer.privacy         = Privacy policy
+footer.termsConditions = Terms and conditions
+footer.govukHelp       = Help using GOV.UK
+
+language.cymraeg = Cymraeg
+language.change.cymraeg = Newid yr iaith i'r Gymraeg
+language.english = English
+language.change.english = Change the language to English
+language.switcher = Language switcher
+
+# Timeout Messages
+# ----------------------------------------------------------
+timeout.title = You’re about to be signed out
+timeout.message = For security reasons, you will be signed out of this service in
+timeout.keepAlive = Stay signed in
+timeout.signOut = Sign out
+
+# BETA Banner
+# ----------------------------------------------------------
+feedback.before = This is a new service - your
+feedback.link = feedback
+feedback.after = will help us to improve it.
+
+# Compliance statement
+# ----------------------------------------------------------
+
+accessibility.title = Accessibility statement for the Job Support Scheme calculator
+## section one
+compliant.one.heading=Accessibility statement for the Eat Out to Help Out Scheme
+compliant.one.lede=This accessibility statement explains how accessible this service is, what to do if you have difficulty using it, and how to report accessibility problems with the service.
+compliant.one.p1.1=This service is part of the wider GOV.UK website. There is a separate
+compliant.one.p1.2=accessibility statement
+compliant.one.p1.3=for the main GOV.UK website.
+compliant.one.p2=This page only contains information about the Job Support Scheme calculator, available at tax.service.gov.uk
+
+## section two
+compliant.two.heading=Using this service
+compliant.two.p1=<To be completed>
+compliant.two.p2=This service is run by HM Revenue and Customs (HMRC). We want as many people as possible to be able to use this service. This means you should be able to:
+compliant.two.l1=change colours, contrast levels and fonts
+compliant.two.l2=zoom in up to 300% without the text spilling off the screen
+compliant.two.l3=get from the start of the service to the end using just a keyboard
+compliant.two.l4=get from the start of the service to the end using speech recognition software
+compliant.two.l5=listen to the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+compliant.two.p3=We have also made the text in the service as simple as possible to understand.
```

Comment:
I assume this temporary as you have English text in the Welsh messages file?

Reply:
yes

Reply:
is it possible to remove this w/o breaking the service?

Reply:
this is needed if we want to show language toggle in the UI,  if you want i can remove all entries and just keep empty file

Reply:
that's what I meant.

Reply:
i thought keep english entries in the file is better than showing just message keys (because of empty file)

Reply:
I would like the only the entries we have require so far in that file. It will make it easier in the long run to just pass that file to the WTT. If we leave all this stuff in and then at the end try to work out what is required and what is not we risk breaking stuff. So for I think we only need the claimStartDate messages in there plus what is required in the header and footer.